azure/image/rhel/variables.pkr.hcl

```diff
@@ -19,23 +19,27 @@ variable "resource_group" {
 }
 
 variable "client_id" {
-  type    = string
-  default = ""
+  type      = string
+  default   = ""
+  sensitive = true
 }
 
 variable "client_secret" {
-  type    = string
-  default = ""
+  type      = string
+  default   = ""
```

Comment:
having empty default value for essential variables wouldn't allow packer to start running? (instead of emitting error at execution)

Reply:
I will add a comment as to why we have these as empty.

Reply:
But just adding comment won't enforce it.